intersection_transit_maneuvering/src/intersection_transit_maneuvering.cpp

```diff
@@ -197,17 +197,19 @@ return CallbackReturn::SUCCESS;
             new_req->maneuver_plan.maneuvers.push_back(i);
         }
 
-            new_req->header = req->header;
-            new_req->vehicle_state = req->vehicle_state;
-            new_req->initial_trajectory_plan = req->initial_trajectory_plan;
-            object_->call(new_req,resp);
```

Comment:
I dont see the "resp" definition being removed after you have renamed it?!

Reply:
resp is the input to the callback, so the definition shouldn't be removed. 
the callback is still using the resp variable in line 212 though from ilc_resp to resp